src/internal/components/pluggableVisualizations/comboChart/PluggableComboChart.tsx

```diff
@@ -95,6 +91,7 @@ export class PluggableComboChart extends PluggableBaseChart {
                 ...COMBO_CHART_UICONFIG_WITH_OPTIONAL_STACKING,
                 optionalStacking: {
                     supported: true,
+                    disabled: this.primaryChartType === VisualizationTypes.LINE,
```

Comment:
please use `isLineChart` util,
it could be `disabled: isLineChart(this.primaryChartType)`

Reply:
fixed